sdk/identity/identity/src/credentials/managedIdentityCredential/fabricMsi.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is the fabric-specific part of the PR just this file and the test below?

If so, I'd be okay with you adding it to the main refactoring PR (if you want)

Reply:
I can do that definitely! Thank you